rimage/manifest.c

```diff
@@ -77,6 +78,21 @@ static int man_open_manifest_file(struct image *image)
 	return 0;
 }
 
+static int man_open_xman_file(struct image *image)
```

Comment:
please use ext_man_ as its easier to follow.

Reply:
changed